examples/on_gke_end_to_end/main.tf

```diff
@@ -59,6 +59,7 @@ provider "helm" {
   debug                           = true
   automount_service_account_token = true
   install_tiller                  = true
+  version = "~> v0.10"
```

Comment:
This has been added as a version constraint within the other modules in a recent PR, I don't believe this is required anymore.